packages/ui/src/data/arranger/formatting.ts

```diff
@@ -0,0 +1,28 @@
+export const MISSING_DATA = '__missing__';
+
```

Comment:
capitalization: I see  (T)rue/(F)alse and (t)rue/(f)alse . Is that wanted?

Reply:
this is arranger idiosyncrasy. I can move them in a dictionnary

Reply:
made a fix, using a dictionnary to regroup specific arrager keys and values